courses/models.py

```
@@ -0,0 +1,137 @@
+    short_introduction = models.CharField(max_length=500)
+    course_description = models.TextField()
+    level = models.CharField(max_length=15, choices=LEVEL_CHOICES, default='beginner')
+
+    course_image = CloudinaryField(
+        folder='course_images/',
+        resource_type="image",
+        overwrite=True,
+        null=True,
+        blank=True
+    )
+    preview_video_link = models.URLField(max_length=200, null=True, blank=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `URLField`

(DJ001)